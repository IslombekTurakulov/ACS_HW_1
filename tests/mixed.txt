functional Funclang1 1924 34 0 1
functional Funclang2 1942 324 1 1
object_oriented object_oriented1 1980  32 1
object_oriented object_oriented2 1991  23 2
object_oriented object_oriented3 1990 242 0
procedural ProcLang3 1951  24 1
procedural ProcLang4 1952 41 0
functional FuncLang3 1954 241 0 1
functional FuncLang4 1952 234 1 1
object_oriented object_oriented4 1980  34 1
object_oriented object_oriented5 1991  32 2
object_oriented object_oriented6 1990  232 0