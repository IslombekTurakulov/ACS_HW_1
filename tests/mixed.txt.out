Input container:
Container contains 12 elements.
0: Functional: name = Funclang1 popularity = 34%, age = 1946, typing type = STRICT, lazy calculation = FALSE age / name.len() = 216.222.
1: Functional: name = Funclang2 popularity = 0%, age = 1946, typing type = STRICT, lazy calculation = TRUE age / name.len() = 216.222.
2: Object Oriented: name = object_oriented1 popularity = 32%, age = 1980, legacy = MULTIPLE age / name.len() = 123.75.
3: Object Oriented: name = object_oriented2 popularity = 23%, age = 1991, legacy = INTERFACE age / name.len() = 124.438.
4: Object Oriented: name = object_oriented3 popularity = 0%, age = 1990, legacy = SINGLE age / name.len() = 124.375.
5: Procedural: name = ProcLang3 popularity = 24%, age = 1951, has abstract variables = FALSE age / name.len() = 216.778.
6: Procedural: name = ProcLang4 popularity = 41%, age = 1952, has abstract variables = TRUE age / name.len() = 216.889.
7: Functional: name = FuncLang3 popularity = 0%, age = 1954, typing type = STRICT, lazy calculation = FALSE age / name.len() = 217.111.
8: Functional: name = FuncLang4 popularity = 0%, age = 1952, typing type = STRICT, lazy calculation = TRUE age / name.len() = 216.889.
9: Object Oriented: name = object_oriented4 popularity = 34%, age = 1980, legacy = MULTIPLE age / name.len() = 123.75.
10: Object Oriented: name = object_oriented5 popularity = 32%, age = 1991, legacy = INTERFACE age / name.len() = 124.438.
11: Object Oriented: name = object_oriented6 popularity = 0%, age = 1990, legacy = SINGLE age / name.len() = 124.375.
